### PR TITLE
Attestation issue #172

### PR DESCRIPTION
**Pull request by anilsingla**
Feature to attest the OSAR

### CODE REVIEW

**Comment by asa1997** [comment]
@anilsingla Please resolve the conflicts.

**Comment by anilsingla** [comment]
resolved conflicts.

File: src/main/bash/commands/besman-help.sh [hunk 1]
```diff
@@ -254,6 +254,38 @@ function __bes_help_run {
...
+    __besman_echo_no_colour '  '
+    __besman_echo_white 'DESCRIPTION'
+    __besman_echo_no_colour '   Used to attest and create the key and signature file for OSAR report.'
+    __besman_echo_no_colour '  '
+    __besman_echo_white 'EXAMPLE'
+    __besman_echo_no_colour '   bes attest --file sonar-scanner.osar'
+    __besman_echo_no_colour '  '
+}
+
+function __bes_help_run {
```
Comment (asa1997) [inline src/main/bash/commands/besman-help.sh:273]: Incorrect function name

**Review by asa1997 — CHANGES_REQUESTED** [review]

**Comment by anilsingla** [comment]
Added path option for the OSAR file.

**Review by asa1997 — APPROVED** [review]
@anilsingla LGTM

**Pull request merged**